react/features/invite/components/add-people-dialog/web/InviteByEmailSection.js

```diff
@@ -95,16 +98,15 @@ function InviteByEmailSection({ inviteSubject, inviteText, t }: Props) {
         return (
             <>
                 {
-                    PROVIDER_MAPPING.map(({ icon, tooltipKey, url }, idx) => (
+                    PROVIDER_MAPPING.map(({ icon, tooltipKey, url, target }, idx) => (
```

Comment:
This fails with `Property `target` is missing in object literal [1].` ...

Reply:
fixed. let's wait before merging this, maybe we get to a better solution